fixed cross icon on set date dialog ccx schedule page

amir-qayyum-khan opened the pull request:
In this PR  I fixed the cross image on set date dialog 

![fix_dialog](https://cloud.githubusercontent.com/assets/10431250/7022279/82ee0988-dd43-11e4-88f2-50498c2533f1.png)
#70

ataki committed:
hard start/stop times to video player

This pull request makes the video player module respect
the video start/stop times specified by the instructor
as a hard start/stops, i.e. the video treats the start
time as the beginning of the video and the stop time as
the end.

Previously, the video module displayed the full
video and displayed a jquery ui slider range representing
the start and end times.

This new functionality invalidates two previous acceptance
tests for the video module:

    - slider visible test
    - finish time video acceptance test.

These tests are removed from the common acceptance tests.

marcotuts committed:
updated styling for LMS post references

ataki committed:
Update variable names for video module

Commit:
Implement profile_image upload and remove endpoints

TNL-1537

Co-Authored-By: Andy Armstrong <andya@edx.org>
Co-Authored-By: cahrens <christina@edx.org>

Commit:
New bio field in UserProfile.

Commit:
Add field that was missing from re-ordered migration

dan-f committed:
Integrate profile images into Accounts API

TNL-1545

symbolist committed:
Account settings page.

TNL-1499

symbolist committed:
Added javascript library backbone-super.

TNL-1499

symbolist committed:
Bokchoy tests for account settings page.

TNL-1499

muhammad-ammar committed:
Jasmine tests for account settings page.

TNL-1499

ataki committed:
Add comment, refactor, handle edge cases

e-kolpakov committed:
Index dictionary for CapaDescriptor + extending tests to cover it

dan-f committed:
Rebase cleanup

dan-f committed:
Add documentation for is_active and profile_image

andy-armstrong committed:
Implement parental controls for user profiles

TNL-1606

dan-f committed:
Implement language proficiencies.

TNL-1488

andy-armstrong committed:
Implement parental controls for the User API

TNL-1739

dan-f committed:
Add logging to image uploads/deletes.

Conflicts:
	openedx/core/djangoapps/profile_images/views.py

e-kolpakov committed:
Filtering using search index + tests

dan-f committed:
Configurations for filesystem and s3 backends.

TNL-1789

Commit:
Updating blue color variables for accessibility in the LMS

symbolist committed:
Enable third party auth from account settings page.

TNL-1535

andy-armstrong committed:
Implement profile image moderation

TNL-1548

Commit:
Link to learner profile page from username in discussions.

TNL-1503

Commit:
Learner profile page.

TNL-1502

marjev committed:
(SOL-531)(SOL-532)Course Cards update; A link to course discovery page has been added to the OpenEdX homepage

Commit:
Remove account settings display and editing from dashboard sidebar.

TNL-1516

andy-armstrong committed:
Add default profile images

TNL-1796

dan-f committed:
Pull additional profile image settings from configuration

andy-armstrong committed:
Bump User API and Profile Images API to v1

TNL-1669

symbolist committed:
Third party auth tests should check account_settings_context().

TNL-1535

Commit:
Include profile image version in URLs.

TNL-1860

muhammad-ammar committed:
Accessibility changes for Account Settings and Learner Profile pages

Commit:
If user is not above age limit, show restricted learner profile.

TNL-1658

Commit:
make auto_auth users satisfy PARENTAL_CONSENT_AGE_LIMIT

Commit:
remove catchall exception handling in profile image api

marcotuts committed:
added images to the bulk email directory to include Youtube as a social icon option

symbolist committed:
Adding functionlity to upload/remove profile image.

TNL-1538

dsego committed:
Dashboard search feature

Commit:
Tab number in url doesn't take to correct tab

When there is multiple units in a particular subsection
then there will be tabs in courseware. When url have
tab number at the end of the url the code was not
considering it and showing the wrong tab. I have changed
such that it will always check the value of position.

TNL-1844

pdpinch commented on lms/templates/ccx/schedule.html:
```diff
@@ -32,7 +32,7 @@
   <div class="inner-wrapper" role="dialog">
     <button class="close-modal">
-      <i class="fa-remove"></i>
+      <i class="fa fa-times"></i>
```
Is that ``fa`` class supposed to be there? What does it do?

Although using an icon named ``fa-times`` confuses me in this case, I've confirmed that it's common practice at edX.

amir-qayyum-arbisoft replied:
fa-time is font awesome class for cross icon, edx is using font awesome , I was not able to find  fa-remove in css files

http://fortawesome.github.io/Font-Awesome/icon/times/

singingwolfboy committed:
Move github dependencies into github.txt

dianakhuang committed:
Add model change event tracking to UserProfile.

e-kolpakov committed:
Added library search indexer and tests
Search indexer is not bound to any library modifications, so tests fail - this is expected.

e-kolpakov committed:
Refactored tests to explicitly state with which modulestores they work

e-kolpakov committed:
Added setting to enable/disable library indexing + moved is_index_enabled into search indexer classes
Added task and signal to fire library reindex

e-kolpakov committed:
Added code to fire library_updated signal when library is updated

bradenmacdonald committed:
Bump problem builder version to latest edx-release build

andy-armstrong committed:
Add client-side events for viewing pages

TNL-1504

nasthagiri committed:
MA-612 Mobile Push Notification Studio Django admin setting

nasthagiri committed:
MA-611, MA-613 Mobile Push Notification Studio Backend; Integration with Parse

awaisdar001 committed:
Fixed Long discussion post not rendering Latex expressions
TNL-1902

Commit:
Bok_choy tests for upload/remove profile image.

TNL-1538

nasthagiri committed:
MA-614 Mobile Push Notification Studio UI

symbolist committed:
Resolving JS lint errors for account settings & learner profile page.

TNL-1855

symbolist committed:
Quality fixes.

Commit:
ECOM-1396: added the function to check is the xblock published

muhammad-ammar committed:
Jasmine tests for upload/remove profile image on learner profile page.

TNL-1538

Shrhawk committed:
Revert hotfix for PLAT-452

andy-armstrong committed:
Limit the size of the user profile's "bio" field

TNL-1990

applecool committed:
Added name to the authors list

sarina committed:
Merge pull request #7674 from applecool/authors

Added name to the authors list

dan-f committed:
Track user settings changed events on User save

nasthagiri committed:
Clean ID of course_module for push notification channel name.
Update Mobile API to include channel_id for course.

e-kolpakov committed:
Fixed broken test

e-kolpakov committed:
Added library update task test

zubair-arbi committed:
show error message to user on photo reverification page if Flash not found
ECOM-1355

e-kolpakov committed:
Moved library key normalization into library_tools module, fixed library update task test

e-kolpakov committed:
Renamed course_key to structure_key to reflect the fact it might be a library key as well.

Commit:
Credit message on track selection page.

* Adds a credit course mode to indicate that a course
has a credit option.

* Hides the credit option from the track selection and
pay-and-verify pages.

* Shows different messaging for the verified track if
it's possible to upgrade from verified to credit at the end
of the course.

Commit:
Moving the country field to the required section of account setting page.

TNL-1856

e-kolpakov committed:
Removed stale test helper method

zubair-arbi committed:
Merge pull request #7679 from edx/zub/bugfix/ecom-1355-ie9-verification-webcam-without-flash

show error message to user on photo reverification page if Flash not fou...

marjev committed:
Merge pull request #7548 from edx/mjevtic/SOL-531

(SOL-531)(SOL-532) Course Cards update on the homepage;

Commit:
ECOM-1396: added the xblock setup in github.txt of edx-platform

adampalay committed:
Merge pull request #7683 from edx/release

Release

cpennington committed:
Sample xblock metrics at 10%

Commit:
Merge pull request #7688 from edx/revert-plat452

Revert hotfix for PLAT-452

mcgachey committed:
[LTI Provider] Create LTI Provider app, initial views and OAuth signature validation

awais786 committed:
ECOM-1420 adding new field and migration file.

ataki committed:
Warning that hard start/stop times don't work for mobile

Commit:
Merge pull request #7669 from edx/will/fix-release-script-escaping

Escape curly quotes in release page table

Commit:
Fixed Embargo Bug in Enrollment API

This issue was not found sooner because the test helper used only tested for GeoIP embargoes, and neglected the user profile. An additional test has been added to test for embargoes against the country assigned to the user's profile. Additionally, variable names have been cleaned up--user replaced with username--to avoid confusion in the future.

andy-armstrong committed:
Event on user preference updates

TNL-1851

singingwolfboy committed:
Remove nested form elements

This is invalid HTML

Commit:
Event change_initiated for password and email change requests.

Commit:
Merge pull request #7666 from edx/gprice/fix-oauth-exchange

Enable OAuth token exchange only if OAuth provider

sarina committed:
Merge pull request #7663 from applecool/edxsprint

Fix for TNL-1639

clintonb committed:
Merge pull request #7692 from edx/ECOM-1418

Fixed Embargo Bug in Enrollment API

muhammad-ammar committed:
Add Simplified and Traditional Chinese as Transcript language options

TNL-1740

e-kolpakov committed:
Added management command to manually reindex libraries

e-kolpakov committed:
Made methods with no implementation abstract

e-kolpakov committed:
Pylint fixes

awais786 committed:
Merge pull request #7690 from edx/awais786/ECOM-1420-new-field

ECOM-1420 adding new field and migration file.

awais786 committed:
ECOM-1421 implementing the functionality

carsongee committed:
Put CCX URLs behind their feature flag
- Moved the CCX URLS into their django app.

waheedahmed committed:
Added new release tag for ORA2.
TNL-2000

Commit:
Returning complete URL for Enrollment API user messaging

The Enrollment API now returns the complete URL in the user_message_url field. Returning just the path can result in 404s if the consuming code simply redirects to the value of this field without first checking to see if the hostname is included. Using the full URL solves this problem for all clients.

Commit:
Increase timeout for library page to load after refresh in acceptance tests. TE-745

adampalay committed:
Revert "Revert "use different queue for smaller emails (TNL-1591)""

This reverts commit 228cca4e38ec79f5cbb54098c788ab9556f602e2.

Commit:
Merge pull request #7665 from edx/rc/2015-04-15

Rc/2015 04 15

singingwolfboy committed:
Simplify test_only_expected_fields_are_displayed

Use sets to accomplish the same result.

Commit:
Expose course start/end date in the enrollment API

Commit:
Expose course start/end date in the enrollment API

cpennington committed:
Fix contract on context argument

Commit:
Merge pull request #7704 from edx/zoldak/TE-745

Increase timeout for library page to load after refresh in acceptance tests

singingwolfboy committed:
Merge pull request #7693 from edx/remove-nested-form-elements

Remove nested form elements

Commit:
Adding outline styles (none) to the sr focus div

Commit:
Chaning table th td color to meet AA guidelines

Commit:
Merge branch 'release'

singingwolfboy committed:
Merge pull request #7111 from ataki/ataki/upstream-pr-video-start-stop

hard start/stop times to video player

singingwolfboy committed:
Enable --pdb debugging

This allows developers to pass the `--pdb` flag to paver test commands,
and that flag will be proxied to the `nosetests` command. It's useful
for debugging purposes.

feanil committed:
Be more specific about where we compile sass from.

There are now folders under static that have sass that should not be compiled
as a part of our asset pipeline,  they were getting pulled in because the wildcard
for `SASS_UPDATE_DIRS` was too broad.  This update makes it more specific for most
folders but leaves it broad for the common folder where the xmodule has its own sass
that needs compiling from a non-standard location.

carsongee committed:
Merge pull request #7700 from mitocw/feature/cg/ccx_urls

Put CCX URLs behind their feature flag

adampalay committed:
Revert "Decrease the number of inserts and updates needed by DjangoKeyValueStore"

This reverts commit 88b918747628a5f6020625e603d4a7184c08583e.

marcotuts committed:
Merge pull request #7221 from edx/marco/discussions-related-to

Updated Styling for course references in discussion forum posts

awaisdar001 committed:
Merge pull request #7649 from edx/aj/tnl1902-fix-long-latex-post-discussion

Fixed Long discussion post not rendering Latex expressions

awaisdar001 committed:
Forgot password model not visible when user clicks 'Forgot Password' link on CMS.

TNL-1921

Commit:
Merge pull request #7544 from edx/ahsan/tnl-1675-iframe-elements-require-meaningful-title

iframe elements require a meaningful title attribute

Commit:
Merge pull request #7682 from edx/aamir-khan/ECOM-1287-reverification-xblock-in-github

ECOM-1396: added the xblock setup in github.txt of edx-platform

Commit:
Merge pull request #7467 from edx/clrux/UX-1810-a11y-color-fix--blue

Accessibility: Updating the color blue on LMS

waheedahmed committed:
Merge pull request #7701 from edx/waheed/tnl2000-ora-release-tag

Added new release tag for ORA2.

feanil committed:
Merge pull request #7715 from edx/feanil/sassy_certs

Be more specific about where we compile sass from.

singingwolfboy committed:
Strict comparison between expected_fields and displayed_fields

adampalay committed:
fix quality violation

andy-armstrong committed:
Isolate Bok Choy tests to unique test users

Commit:
Fixed caret position for LTR and RTL (SOL-693)

Commit:
Emit events for language proficiencies from the update_account_settings method.

Commit:
Fixed z-index for social action dropdown (TNL-1806)

Commit:
Fixing the SaSS for RTL.

TNL-1759

adampalay committed:
Merge pull request #7716 from edx/hotfix/2015-04-16

Hotfix/2015 04 16

Commit:
Better i18n (cont.)

1. Fix sentence breaks inside `photo_reverification.html` and `midcourse_photo_reverification.html`.
2. Combine three setences inside `choose.html`.
3. Adding some punctuation marks to reduce duplicated strings with only different Punctuation marks in translation.
4. Move the colons inside the strings to be translated. This may be not necessary for now, but I think this may help translation by removing potential word meaning conflict.
5. Sorry for

Commit:
DropDownFieldView should show empty option if model value is unset.

TNL-2022

adampalay committed:
Merge pull request #7724 from edx/release

Release

Commit:
Set default profile visibility to all_users.

sarina committed:
Merge pull request #7670 from louyihua/lms_i18n_1

Better i18n (cont.)

muhammad-ammar committed:
Fixed gettext references.

TNL-1862

Commit:
Tests for profile image eventing.

Commit:
Merge pull request #7712 from edx/clrux/UX-2001-summary-table-contrast

Chaning table th td color to meet AA guidelines

adampalay committed:
reset queue reference in aws.py, use low priority queue (TNL-1591) (TNL-2003)

Commit:
Pep8/pylint error.

marcotuts committed:
moved the dropdown actions outside of the if show_courseware_link in attempt to still show the dropdown actions

marcotuts committed:
Merge pull request #7608 from indagation/close-modal

Fix modal close button

andy-armstrong committed:
Code review feedback and test cleanup.

bradenmacdonald committed:
Merge pull request #7638 from open-craft/problem-builder-version-bump

Bump version of Problem Builder

awais786 committed:
Merge pull request #7698 from edx/awais786/ECOM-1421-modify-course-access-api

ECOM-1421 implementing the functionality

symbolist committed:
Removed all references to AUTH_ENTRY_DASHBOARD.

TNL-2024

e-kolpakov committed:
Tests for reindex library management command

e-kolpakov committed:
Removed unnecessary comments

Commit:
Changing id to class

mhoeber committed:
Update component templates for accessibility and content

andy-armstrong committed:
Obey learner profile default visibility

cewing committed:
Add Cris Ewing and Carlos de La Guardia to the authors list

Both contributed to the MIT CCX pull request

sarina committed:
Merge pull request #7738 from jazkarta/cewing-add-contributor

Add Cris Ewing and Carlos de La Guardia to the authors list

marcotuts committed:
Merge pull request #7730 from edx/marco/fix/course-refund-via-unenroll

Still show dropdown actions even if course isn't accessible.

e-kolpakov committed:
Merge pull request #7641 from open-craft/eugeny/problem-type-search

Capa problem type filtering using edx-search

muhammad-ammar committed:
Update user message strings

Commit:
Merge pull request #7687 from edx/will/ecom-1408-take-two

Credit message on track selection page.

awais786 committed:
ECOM-1439 adding new model and migration.

Commit:
Merge pull request #7711 from edx/clrux/UX-1946-div-focus-fix

Adding outline styles (none) to the sr focus div

cewing committed:
MIT CCX: Fix display of CCXs on dashboard

repairs https://openedx.atlassian.net/browse/TNL-2007 and https://openedx.atlassian.net/browse/TNL-2009

Original Commit Messages:

Update ccx dashboard display to match new course display

Fix display of CCX dates on dashboard to use dates from the CCX instead of the related course.

The fix is implemented by providing date- and course-related APIs on the CCX object itself.  These APIs are mimicked from the same APIs available on a course descriptor.

bring the methods for printing start and end datetimes into line with course by adding UTC under the proper circumstances

add test coverage for new APIs defined on the CCX model

fix quality violation

sarina committed:
Upgrade Django 1.4 to latest point release

cpennington committed:
Merge pull request #7686 from cpennington/xmodule-metrics-sample-rate

Sample xblock metrics at 10%

Commit:
Merge pull request #7721 from edx/clrux/UX-1953-dropdown-fix

Fixed caret position for LTR and RTL (SOL-693)

Commit:
Merge pull request #7708 from edx/will/ecom-1411

Expose course start/end date in the enrollment API

Commit:
Revert "Expose course start/end date in the enrollment API"

alawibaba committed:
Added salt to "load" beacon.

johnjlee committed:
Update schoolyourself-xblock to the latest version, which adds the ability to set a partner ID and updates the CSS to fix the student_view things on Firefox

Commit:
Merge pull request #7743 from edx/revert-7708-will/ecom-1411

Revert "Expose course start/end date in the enrollment API"

adampalay committed:
Merge pull request #7729 from edx/adam/different-queue-small-emails

Adam/different queue small emails

alawibaba committed:
Merge pull request #7744 from edx/multibeacon

Added salt to "load" beacon.

clintonb committed:
Merge pull request #7737 from edx/enrollment-api-full-urls

Returning complete URL for Enrollment API user messaging

andy-armstrong committed:
Merge pull request #7350 from edx/learner-profiles

Feature branch for learner profiles

nasthagiri committed:
Merge pull request #7676 from edx/mobile/notifications-backend

Mobile/notifications

awaisdar001 committed:
Fixed change email address error from student dashboard
TNL-1745

symbolist committed:
Accessibility tweaks for account settings and profile pages.

TNL-1660

muhammad-ammar committed:
Merge pull request #7696 from edx/ammar/tnl1740-add-simplified-and-traditional-chinese-transcript-language-options

Add Simplified and Traditional Chinese as Transcript language options

Commit:
Mark test_split_test_LMS_staff_view as flaky

cpennington committed:
Merge pull request #7710 from cpennington/relax-context-contract

Fix contract on context argument

sarina committed:
Update translations (autogenerated message)

sarina committed:
Merge pull request #7758 from edx/sarina/20150421-translations-update

Update translations (autogenerated message)

Commit:
Merge pull request #7756 from edx/will/ecom-1408-take-two

Expose course start/end date in the enrollment API

Commit:
Merge pull request #7754 from edx/usman/tnl1660-fixes

Accessibility tweaks for account settings and profile pages.

Commit:
Added Payment Cancellation Page

This commit creates a new page intended only for use when the user cancels payment. The previous implementation relied on the CyberSource callback page, which is not useful for orders placed with the E-Commerce Service.

singingwolfboy committed:
_parse_time_str can be a standalone function

Commit:
RTL-ifying the due icon

Commit:
RTL-ifying the sidebar title in Handouts

Commit:
Don't event first_name and last_name.

TNL-2044

Commit:
RTL-ifies the sequence nav buttons

talbs committed:
fix: resolving edx.org and Open edX breakpoint clash
* simplifying edx.org breakpoints to use min-width
* creating -bp-* centric Sass variables
* converting all existing edx.org centric media queries to use new Sass variables

ECOM-1457

Commit:
Use ecommerce api v2.

XCOM-213 XCOM-214

Commit:
Merge pull request #7769 from edx/christina/eventing_fix

Don't event first_name and last_name.

clintonb committed:
Merge pull request #7762 from edx/commerce-checkout-cancel

Added Payment Cancellation Page

awais786 committed:
Merge pull request #7740 from edx/awais786/ECOM-1439-skip-icrv-model

ECOM-1439 adding new model and migration.

Commit:
Merge pull request #7766 from edx/clrux/UX-1634-sidebar-header-rtl

RTL-ifying the sidebar title in Handouts

talbs committed:
Merge pull request #7770 from edx/talbs/rc/2015-04-22/fix-breakpoints

Fix: edx.org and Open edX breakpoint clash - ECOM-1457

singingwolfboy committed:
Merge pull request #7705 from edx/adjust-test_only_expected_fields_are_displayed

Simplify test_only_expected_fields_are_displayed

sarina committed:
Merge pull request #7741 from edx/sarina/upgrade-django1.4.18-20

Upgrade Django 1.4 to latest point release

singingwolfboy committed:
Update paver tests to ignore trailing whitespace

singingwolfboy committed:
Merge pull request #7764 from edx/parse-time-as-standalone-function

_parse_time_str can be a standalone function

Commit:
Add logging for verification and reverification.

Adds additional logging for the initial/re-verification flows.
This should make it easier to understand the behavior of
the production system and provide a way to identify issues early.

singingwolfboy committed:
Remove extra spaces from paver commands

Commit:
Merge branch 'release'

Commit:
Merge pull request #7768 from edx/clrux/UX-1724-RTL-sequence-nav-buttons

RTL-ifies the sequence nav buttons

Commit:
tweak firefox upgrade instructions

symbolist committed:
Change image upload input name and not make duplicate of file in data.

TNL-2052

singingwolfboy committed:
Merge pull request #7714 from edx/enable-pdb-debugging

Enable --pdb debugging

andy-armstrong committed:
Fix default profile images for edx.org

TNL-2054

dianakhuang committed:
Merge pull request #7784 from edx/andya/hotfix-default-profile-image

Fix default profile images for edx.org

dianakhuang committed:
Merge pull request #7783 from edx/hotfix/2015-04-22

Change image upload input name and not make duplicate of file in data.

dianakhuang committed:
Merge pull request #7786 from edx/release

Merge release back into master

applecool committed:
Fix to LOC-66

Commit:
Merge pull request #7660 from edx/aamir-khan/ECOM-1396/checkoint-name-field

Aamir khan/ecom 1396/checkoint name field

zubair-arbi committed:
fix reverification link in lms
ECOM-1437

awaisdar001 committed:
Merge pull request #7718 from edx/aj/tnl1921-forgot-password-model-not-showing

Forgot password model not visible when user clicks 'Forgot Password' link on CMS.

zubair-arbi committed:
Merge pull request #7774 from edx/zub/story/ecom-1437-reverification-lms-fixes

fix reverification link in lms

sarina committed:
Merge pull request #7667 from applecool/L10N_bugfix

Fix to LOC-66

mhoeber committed:
Updated enrollment api doc for new response fields

And cleaned up one doctoring

singingwolfboy committed:
Merge pull request #7623 from edx/move-github-reqs-to-github.txt

Move github dependencies into github.txt

Commit:
PEP8 needs to ignore .git

cewing committed:
MIT CCX: Code Quality

refs https://openedx.atlassian.net/browse/TNL-2009

nedbat committed:
Merge pull request #7793 from edx/ned/pep8-needs-to-ignore-dotgit

PEP8 needs to ignore .git

sarina committed:
Update i18n tools to the latest version

sarina committed:
Merge pull request #7795 from edx/sarina/update-i18ntools

Update i18n tools to the latest version

bradenmacdonald committed:
Bump problem_builder version to fix DB table conflict

cewing committed:
MIT CCX: Respond to PR feedback

Use the lazy module to create cached attributes

Ensure tests for localized date formatting are insulated from changes in default formatting

Other minor fixes

Commit:
Merge pull request #7689 from mcgachey/mcgachey-lti

[LTI Provider] Create LTI Provider app, initial views and OAuth signature validation

nedbat committed:
Merge pull request #7771 from jazkarta/mit-ccx-fix-dashboard

WIP: MIT CCX student dashboard display

mhoeber committed:
Merge pull request #7791 from edx/markhoeber-doc1904

Updated enrollment api doc for new response fields

Commit:
Merge pull request #7587 from edx/waqas/tnl1844-wrong-tab-problem

Tab number in url doesn't take to correct tab

nasthagiri committed:
MA-668 Mobile Push Notification payload update
Support iOS receiving the update even when in the background.
Support Android detecting duplicates.

Commit:
Merge pull request #7624 from mcgachey/mcgachey-courseware-flags

[LTI Provider] Adding options to suppress chrome for embedding courseware

bradenmacdonald committed:
Merge pull request #7796 from open-craft/fix-mentoring-answers-conflict

Fix for DB table conflict preventing DB update on fresh installs

sarina committed:
Update translations (autogenerated message)

Commit:
Merge pull request #7765 from edx/clrux/UX-1632-due-icon-rtl-placement

RTL-ifying the due icon

sarina committed:
Only pull languages we want from Transifex

Kelketek committed:
Bump version of problem builder

+1s available in https://github.com/open-craft/problem-builder/pull/19

Kelketek committed:
Merge pull request #7801 from edx/problem-builder-bump

Bump version of problem builder

nedbat committed:
Merge pull request #7745 from schoolyourself/schoolyourself-xblock-update

Update schoolyourself-xblock to the latest version

sarina committed:
Merge pull request #7800 from edx/sarina/20150424-update-translations

Update translations & how translations are pulled

Commit:
Merge pull request #7781 from edx/jsa/bokchoy-firefox-instrux

tweak firefox upgrade instructions

Commit:
Skip unreliable test. TNL-2035

Commit:
Merge pull request #7775 from edx/will/verification-logging

Add logging for verification and reverification.

Commit:
Merge pull request #7522 from edx/feature/dashboard-search

Feature/dashboard search

Commit:
Merge pull request #7806 from edx/benp/skip-test-TNL-2035

Skip unreliable test. TNL-2035

nasthagiri committed:
Merge pull request #7805 from edx/mobile/push-notification-ios

MA-668 Mobile Push Notification payload update

mhoeber committed:
Merge pull request #7579 from edx/sylvia/docs/templateupdates

Update component templates, round 1

Commit:
Merge pull request #7777 from edx/jsa/baskets

use commerce api v2.

awaisdar001 committed:
Merge pull request #7753 from edx/aj/tnl1745-change-email-on-dashboard

Fixed change email address error from student dashboard

Commit:
fixed cross icon issue on set date dialog inside schedule tab

pdpinch commented:
This was merged upstream in https://github.com/edx/edx-platform/pull/7831
